compiler/luci/pass/src/ReplaceNonConstFCWithBatchMatMulPass.cpp

```diff
@@ -150,12 +146,16 @@ bool replace_fc_with_matmul(luci::CircleFullyConnected *fc)
     return ac;
   };
 
-  auto bc = dynamic_cast<luci::CircleConst *>(b);
-  if ((nullptr != bc) && !all_zero(bc))
+  auto bc = dynamic_cast<luci::CircleConst *>(fc->bias());
+  // NOTE bias can be empty as CircleOutputExclude type
+  // NOTE we can only handle bias as FLOAT32 type as of now
+  if (nullptr != bc && bc->dtype() != loco::DataType::FLOAT32)
+    return false;
```

Comment:
CircleBatchMatMul is already created at this point (which changes the graph). 

It would be better to move this code before line 130.